index.dd

```diff
@@ -95,12 +95,13 @@ $(DIVC boxes,
         )
         $(TOUR comments, Community,
             $(P Discuss D on the $(LINK2 https://forum.dlang.org/, forums), join
-                the $(LINK2 irc://irc.freenode.net/d, IRC channel), or follow
-                us on $(LINK2 https://twitter.com/D_Programming, Twitter).)
+                the $(LINK2 irc://irc.freenode.net/d, IRC channel), read our
+                $(WEB https://dlang.org/blog, official Blog) or follow us
```

Comment:
Link is broken (`WEB` implies "http://").

I don't know if we demand the Oxford comma, but if we do, one's missing here before "or".